src/liballoc/linked_list.rs

```diff
@@ -1071,6 +1071,15 @@ impl<'a, T, F> Iterator for DrainFilter<'a, T, F>
     }
 }
 
+#[stable(feature = "drain_filter_drop", since = "1.25.0")]
```

Comment:
@sfackler While trait impls are insta-stable, the type it is implemented for is still unstable, so we can still keep this `#[unstable]` (see the `Debug` impl below).

Furthermore, 1.25 doesn't branch off until January, so even if we keep this `#[stable]` the version should still be 1.24.

Reply:
Ah right, thanks for the correction! Sorry about that, @tmccombs - can you switch it back to unstable?

Reply:
haha sure